tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,36 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## March 28, 2023
+
+**General changes**
+
+- Add 2 RCUs, 4 RCUs, and 8 RCUs specifications for [changefeeds](tidb-cloud/changefeed-overview.md), and support choosing your desired specifications when you [create a changefeed](tidb-cloud/changefeed-overview#create-a-changefeed.md).
```

Comment:
If users can choose only one specification each time, how about using the singular form?
```suggestion
- Add 2 RCUs, 4 RCUs, and 8 RCUs specifications for [changefeeds](tidb-cloud/changefeed-overview.md), and support choosing your desired specification when you [create a changefeed](tidb-cloud/changefeed-overview#create-a-changefeed.md).
```